server/index.js

```diff
@@ -7,19 +7,21 @@ app.use(express.static('client'));
 
```

Comment:
If you're using process.env with your port, you should have require/config env statement at the top of this file.